ui/selectors/selectors.js

```diff
@@ -564,8 +540,8 @@ export function getAccountName(accounts, accountAddress) {
 
 export function getMetadataContractName(state, address) {
   const tokenList = getTokenList(state);
-  const entry = Object.values(tokenList).find((identity) =>
-    isEqualCaseInsensitive(identity.address, address),
+  const entry = Object.values(tokenList).find((internalAccount) =>
```

Comment:
Are tokens considered accounts? Feels like this one a is a wrong "find & replace"? I would recommend renaming it to `token` instead of `internalAccount` WDYT?

Reply:
Yea, that makes sense.